doc/Install.md

```diff
@@ -249,10 +250,20 @@ You can generally find the location of PHP by executing "which php".
 If you run into trouble with this section please contact your hosting provider for assistance.
 Friendica will not work correctly if you cannot perform this step.
 
-If it is not possible to set up a cron job then please activate the "frontend worker" in the administration interface.
+#### worker alternative: daemon
+Otherwise, you’ll need to use the command line on your remote server and start the Friendica daemon (background task) using the following command:
+
+    cd /path/to/friendica; php bin/daemon.php start
+Once started, you can check the daemon status using the following command:
+
+    cd /path/to/friendica; php bin/daemon.php status
+
+After a server restart or any other failure, the daemon needs to be restarted. This could be achieved by a cronjob.
```

Comment:
Actually, please add a line feed before each new sentence (after the period). This is an internal convention to make diffs nicer when editing Markdown files.